bundler/spec/support/builders.rb

```diff
@@ -35,6 +35,11 @@ def build_repo1
       build_repo gem_repo1 do
         FileUtils.cp rake_path, "#{gem_repo1}/gems/"
 
+        build_gem "coffee-script-source"
+        build_gem "git"
+        build_gem "puma"
+        build_gem "minitest"
```

Comment:
I added those gems to make the "remove specs" pass with minimal changes. 

We can replace them with existing gems like `foo`, `thin`, and so on. What do you think?